radis/spectrum/spectrum.py

```diff
@@ -541,7 +544,21 @@ class directly. Ex::
                 kwloadtxt[k] = kwargs.pop(k)
         argsort = kwargs.pop("argsort", False)
 
-        w, I = np.loadtxt(file, **kwloadtxt).T
+        try:
+            data = np.loadtxt(file, **kwloadtxt)
+        except Exception as err:
+            # invalid format ; expected (w,I)
+            print(f"Invalid format in {file} ({str(err)}). Trying with pandas.read_csv")
+            data = pd.read_csv(file, **kwloadtxt).values
+        if data.shape[0] != 2 and data.shape[1] != 2:
+            # invalid format ; expected (w,I)
+            raise ValueError(
+                f"Invalid format in {file} : {data.shape} (see first line : {data[0]}). Expected two arrays (w, I)"
+            )
+        elif data.shape[0] != 2 and data.shape[1] == 2:
```

Comment:
@erwanp I pull your PR, was trying to use `load_txt` using `pd.read_csv` and it wasn't able to load it. Can you give me some example where this will work?

Reply:
Can you share a small sample or the file you tried to load?

Reply:
```
from radis import Spectrum, SpectrumFactory
wmin = 9000
wmax = 10000
dv = 0.005
T = 3000.0 #K
p = 0.1 #bar
broadening_max_width=10

sf = SpectrumFactory(wavenum_min=wmin, wavenum_max=wmax, 
                  pressure=p,
                  wstep=dv,
                  broadening_max_width=broadening_max_width, 
                  molecule="CO",
                  cutoff=0, # 1e-27,
                  verbose=3,
                  )

sf.fetch_databank('hitemp')

sf.params['optimization'] = None
sf.params['broadening_method'] = 'voigt'
s_none = sf.eq_spectrum(T)

s_none.savetxt('test_save.txt', 'radiance_noslit', wunit='nm', Iunit='W/cm2/sr/µm')

txt =  Spectrum.from_txt('test_save.txt', 'radiance',
                               waveunit='nm', unit='mW/cm2/sr/nm')
```

Reply:
The test_save.txt is like:
```
# Wavelength [air] (nm)	radiance_noslit (W/cm2/sr/Âµm)
1.110806894097350323e+03 5.148886883066076228e-11
1.110806276981198607e+03 5.580955858087338981e-11
1.110805659865732423e+03 5.952810369995782760e-11
1.110805042750951998e+03 6.271845154098717948e-11
1.110804425636857331e+03 6.553906156696676372e-11
1.110803808523448197e+03 6.821177748848249038e-11
1.110803191410724821e+03 7.091618729142113399e-11
1.110802574298686977e+03 7.376849513976415669e-11
1.110801957187335120e+03 7.670531709416791189e-11
.............
.............
.............
```